language/grounding/intro-grounding.ipynb

```diff
@@ -0,0 +1,743 @@
+{
```

Comment:
<p>Nit: Not sure what the proper format is, but I prefer putting text-bison and chat-bison in code font</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Fixed in b15bbec. Thanks!